sphinxext/opengraph.py

```diff
@@ -209,4 +219,3 @@ def setup(app: Sphinx) -> Dict[str, Any]:
         "parallel_read_safe": True,
         "parallel_write_safe": True,
     }
-
```

Comment:
no need to delete, eol @ eof

Reply:
Has been merged, https://github.com/wpilibsuite/sphinxext-opengraph/pull/16 to hotfix